src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -42,17 +53,46 @@ const SendTilBeslutterFooter: React.FC<{
 }) => {
     const { axiosRequest } = useApp();
     const navigate = useNavigate();
-    const { hentTotrinnskontroll, hentBehandling, hentBehandlingshistorikk } = useBehandling();
+    const { behandlingstype, hentTotrinnskontroll, hentBehandling, hentBehandlingshistorikk } =
+        useBehandling();
     const [laster, settLaster] = useState<boolean>(false);
     const [feilmelding, settFeilmelding] = useState<string>();
     const [visModal, settVisModal] = useState<boolean>(false);
+    const [kanOppretteFremlegg, settKanOppretteFremlegg] = useState<boolean>(false);
+    const [skalOppretteFremleggsoppgave, settSkalOppretteFremleggsoppgave] = useState<
+        FremleggsoppgaveType[]
+    >(Object.values(FremleggsoppgaveType));
```

Comment:
Hvorfor instansieres staten for hvilke fremleggsoppgaver som skal opprettes, med en liste som allerede består av verdier egentlig?

Reply:
Default true